src/components/customizableAside.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import CustomizableAvatar from "./components/common/customavatar/CustomizableAvatar"
```

Comment:
@Sheyitofunmi The path to `CustomizableAvatar` that you are using seems incorrect. This path works for me:

```js
import CustomizableAvatar from "../components/common/customavatar/CustomizableAvatar";
```